src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -426,6 +426,9 @@ export default {
 			if (this.sendingErrorCanRetry) {
 				return t('spreed', 'Failed to send the message. Click to try again')
 			}
+			if (this.sendingFailure === 'quota') {
+				return t('spreed', 'Could not upload due to insufficient storage')
```

Comment:
* Not enough free space to upload "{fileName}"

Reply:
dammit... my focus isn't the same lately...